I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/freeroam.rpy", line 489, in script call
    call eventpicker
  File "game/lifeevents.rpy", line 118, in script
    $ rng1 = renpy.random.randint (1,4)
Exception: Possible infinite loop.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/freeroam.rpy", line 489, in script call
    call eventpicker
  File "game/lifeevents.rpy", line 118, in script
    $ rng1 = renpy.random.randint (1,4)
  File "C:\renpy-8.0.2-sdk\renpy\execution.py", line 61, in check_infinite_loop
    raise Exception("Possible infinite loop.")
Exception: Possible infinite loop.

Windows-10-10.0.22621 AMD64
Ren'Py 8.0.2.22081402
Life of Meta 1.0
Thu Mar 16 13:26:16 2023
